submissions/israelOladosu/php-db/classes/dbh.class.php

```diff
@@ -0,0 +1,26 @@
+<?php 
+// class responsible for connecting to the Db
+class Dbh {
+
+    private $host = "localhost";
+    private $user = "root";
+    private $pwd = "mysql";
+    private $dbName = "cookiesDb";
```

Comment:
Is this the name of the Database you created for your e-commerce task?

Reply:
yes...